app.js

```diff
@@ -23,7 +23,7 @@ client.on('ready', () => {
 client.on('guildCreate', createMutedRole);
 
```

Comment:
Would it be better to move this creation of the role to the listener for when the bot reboots (perhaps the ready event)? If so, we would not need to remove the bot from the main server in order to create the new role. We should also make sure to not create a new role with same name if there is already such a role.

Reply:
On hold, may not implement as this would result in `createMutedRule()` running every time the bot reboots.